src/EditorFeatures/CSharp/CSharpEditorFeatures.csproj

```diff
@@ -13,7 +13,6 @@
   <ItemGroup Label="Project References">
     <ProjectReference Include="..\..\Compilers\Core\Portable\CodeAnalysis.csproj" />
     <ProjectReference Include="..\..\Compilers\CSharp\Portable\CSharpCodeAnalysis.csproj" />
-    <ProjectReference Include="..\..\Workspaces\Core\Desktop\Workspaces.Desktop.csproj" />
```

Comment:
so Workspace.Desktop is gone? what happen to ones that are not related to msbuild workspace but in desktop projecT?

Reply:
I see. so now all these things in Desktop is accessed by MEF.

Reply:
Workspaces.Desktop no longer contains MSBuildWorkspace. This will allow us to eventually make Workspaces.MSBuild target netstandard2.0.